variables.tf

```diff
@@ -42,6 +42,10 @@ variable "global_region" {
   description = "OVH global location for PostGreSQL and VPC"
   type        = string
 }
+variable "region_s3" {
```

Comment:
To be deleted

Reply:
That's tricky : for s3, region needs to be lower case whereas upper case is needed for GLOBALE_REGION. `See local.env.template.sh`

Reply:
see provider.tf : 
region = lower(var.global_region)